sql_grader/models.py

```diff
@@ -165,98 +164,3 @@ def create_database(database_name):
     contents = resource_string(pathname)
     database = SqlProblem.create_database_from_sql(contents)
     return database
-
-
-# Memoize the seed databases
-# to avoid continually loading from disk
-DATABASES = {
```

Comment:
Ironically, this was added for perf reasons, to prevent needing to access the filesystem/hdd on each request (to load the `.sql` file).

I wonder if this contributed to the memory issues?

In assuming it's not also a perf increase (I assume it's fine), I think it's fine to spin up the actual sqlite-db-object inside codejail per request, but it'd be ideal to at least read and cache the file contents at run time. Disk access can be slow.